resources/styles/molecules/testimonial/index.scss

```diff
@@ -0,0 +1,131 @@
+@use "@imarc/pronto/resources/styles/imported" as pronto;
+
+.testimonial {
+    $b: &;
+
+    display: grid;
+    row-gap: 1rem;
+    column-gap: 1rem;
+    margin-left: 0;
+    padding-left: 0;
+    box-shadow: none;
+
+    &__content {
+        grid-row: 1 / 2;
```

Comment:
I don't think `grid-row`, `font-size`, or `color: var(--root-color)` are doing anything on this class.